glean-core/python/glean/_loader.py

```diff
@@ -90,6 +90,44 @@ def __getattr__(self, attr):
         )
 
 
+def event_extra_factory(name, argnames):
```

Comment:
Probably should be `_event_extra_factory` to make it "private" (by convention).  Also add type hints.
```suggestion
def _event_extra_factory(name: str, argnames: List[str]) -> EventExtraClass:
```